apis/v1beta1/vspheremachinetemplate_webhooks_test.go

```diff
@@ -54,7 +55,8 @@ func TestVSphereMachineTemplate_ValidateCreate(t *testing.T) {
 	}
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
-			err := tc.vsphereMachine.ValidateCreate()
+			webhook := &VSphereMachineTemplateWebhook{}
+			err := webhook.ValidateCreate(context.TODO(), tc.vsphereMachine)
```

Comment:
```suggestion
			err := webhook.ValidateCreate(context.Background(), tc.vsphereMachine)
```
nit: context.TODO() is usually used in places where at some point a real ctx should be introduced (usually as soon as a ctx is passed into the current func).

Given that we absolutely don't care about it here I would use `context.Background()` in cases like this instead

P.S. context.TODO is meant in a sense of that it's a TODO to replace it at some point and folks can grep for context.TODO.